README.md

```diff
@@ -72,6 +72,7 @@ jobs:
 | git-push-sign-off | If true it will sign-off commit | `false` | false |
 | git-push-user-email | If empty the no-reply email of the GitHub Actions bot will be used (i.e. `github-actions[bot]@users.noreply.github.com`) | `""` | false |
 | git-push-user-name | If empty the name of the GitHub Actions bot will be used (i.e. `github-actions[bot]`) | `""` | false |
+| git-sub-dir | Name of subdirectory if if checkout `path` option is used (i.e. `modules`) | `""` | false |
```

Comment:
```suggestion
| git-sub-dir | Name of subdirectory if checkout `path` option is used (i.e. `modules`) | `""` | false |
```

Reply:
Thank you. Corrected and pushed a squashed change so when this one merges there's only one commit